.github/workflows/docs-pr.yaml

```diff
@@ -6,6 +6,9 @@ on:
       - "docs/**"
     branches:
       - master
+  push:
+    paths:
+      - "docs/**"
```

Comment:
Do we want this?

Reply:
I think we do.  So I'll move it to another PR and ask you and @nopcoder to TAL.

Reply:
Removed this, opened #6311.